src/app/directives/capitalize.directive.ts

```diff
@@ -0,0 +1,10 @@
+import { Directive, ElementRef } from '@angular/core';
+
+@Directive({
+  selector: '[capitalize]',
+})
+export class CapitalizeDirective {
+  constructor(private element:ElementRef) {
+    this.element.nativeElement.style.textTransform = "capitalize";
```

Comment:
Esta implementación solo cambia visualmente el texto, no el valor del elemento, favor de implementarlo de forma que si se transforme el texto.

Reply:
I created an util function to capitalize all the text.